backend/gamify/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
-# Create your models here.
+
+class Team(models.Model):
+    name = models.CharField(max_length=100, blank=False, null=False)
+    join_code = models.CharField(max_length=100, blank=False, null=False)
+
+
+class User(models.Model):
```

Comment:
can we also have a constructor for this that defaults to just a name with no team

Reply:
Doesn't Django's built in constructor already do that b/c the other two fields are nullable?

Reply:
oh yeah maybe, cool nvm